### PR TITLE
chore: fix readme

### PR DESCRIPTION
**Pull request by nikhilmakhijani**
Updating variable description for random_project_id to explain the use case and why we need it. Issue no #723

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@nikhilmakhijani
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_documentation
 The documentation needs to be regenerated. Please run `make generate_docs`. 
 ```bash 
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' '--exclude=autogen' '--exclude=*.tfvars' /workspace/README.md /tmp/tmp.rBzzPOrPPh/generate_docs/workspace/README.md
152c152
< | random\_project\_id |  Adds a suffix of 4 random characters to the `project_id`. It is important to note that once you delete a project you can't use the same `project_id` for 30 days. `random_project_id` comes in handy in such situations (specially during testing) since it allows you to create and shut down projects without worrying about conflicts. | `bool` | `false` | no |
---
> | random\_project\_id | Adds a suffix of 4 random characters to the `project_id`. It is important to note that once you delete a project you can't use the same `project_id` for 30 days.<br>  `random_project_id` comes in handy in such situations (specially during testing) since it allows you to create and shut down projects without worrying about conflicts. | `bool` | `false` | no |
Error: Documentation generation has not been run, please run the
'make docker_generate_docs' command and commit the above changes.
 ```

**Comment by comment-bot-dev** [comment]
@nikhilmakhijani
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR @nikhilmakhijani
Lets move the rationale to our [FAQ](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/docs/FAQ.md)

**Comment by comment-bot-dev** [comment]
@nikhilmakhijani
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_whitespace
 Failed whitespace check. More details below. 
 ```bash 
Checking for trailing whitespace
Checking for missing newline at end of file
Error: No newline at end of file ./variables.tf
Error: No newline at end of file ./README.md
 ```

**Comment by comment-bot-dev** [comment]
@nikhilmakhijani
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by ericsampson** [comment]
@bharathkkb can this be reopened? It would still be good information to add (with the update regarding project id behavior after deletion that you mentioned).

File: docs/FAQ.md [hunk 1]
```diff
@@ -83,3 +83,7 @@ Therefore, the Project Factory deletes the default Service Account to prevent th
 1. No default roles. This Service Account doesn't have access to any GCP resources unless you explicitly grant them.
 
 2. No default usage. With the default Service Account deleted, you have to be explicit in choosing a Service Account for VMs which ensures developers make an informed choice when deciding what access level to give applications.
+
+## What is the requirement of `random_project_id` variable?
```
Comment (g-awmalik) [inline docs/FAQ.md:87]:
```suggestion
## Requirement for `random_project_id`
```

**Review by g-awmalik — CHANGES_REQUESTED** [review]

**Review by g-awmalik — APPROVED** [review]

**Pull request merged**